File: c#/abstractfactory/Ejm1/LuxuryVehicleFactory.cs

namespace AbstractFactory.Ejm1
{
    public class LuxuryVehicleFactory : VehicleFactory
    {
        public override Car getCar()
        {
            return new LuxuryCar("L-C");
        }

        public override SUV getSUV()
        {
            return new LuxurySUV("L-S");
        }
    }
}

File: java/abstractfactory/ej1/LuxuryVehicleFactory.java
package patrones.abstractfactory.ej1;

public class LuxuryVehicleFactory extends VehicleFactory {
	public Car getCar() {
		return new LuxuryCar("L-C");
	}

	public SUV getSUV() {
		return new LuxurySUV("L-S");
	}
}
